recipes/sml/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright (c) 2016-2020 Kris Jusiak (kris at jusiak dot net)
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+#include <boost/sml.hpp>
+#include <cassert>
+
+namespace sml = boost::sml;
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#minimalistic-source-code

Is there any way we can make a cross platform test package? it does not need to do anything just use those headers should be sufficient?

Reply:
@krzysztof-jusiak, could you please share a simpler cross-platform test example?